src/parsers.rs

```diff
@@ -144,6 +144,19 @@ named!(pub parse_u32s<Vec<u32> >, separated_list!(space, complete!(parse_u32)));
 /// Parses a sequence of whitespace seperated i32s.
 named!(pub parse_i32s<Vec<i32> >, separated_list!(space, parse_i32));
 
+/// Parses a bit into a boolean
+named!(pub parse_bit<bool>,
```

Comment:
I think this would be simpler using the alt! combinator, like so: 

```rust
named!(pub parse_bit<bool>, alt!(
          char!('0') => { |_| false }
        | char!('1') => { |_| true }
));
```